First iteration of sase series (#20255)
cloudflare/cloudflare-docs: src/content/docs/learning-paths/sase-overview-course/series/video-1.mdx

Reviewer: URL paths are also relevant for SEO. I'd rename the files `video-1.mdx`, etc. to something more meaningful, such as `evolution-corporate-networks.mdx` for this particular file.

Author: done

---
pcx_content_type: navigation
title: The evolution of corporate networks
sidebar:
  order: 2
tableOfContents: false
---

import { Render, Tabs, TabItem, Stream, Card } from "~/components";

<Tabs>
  <TabItem label="Watch this episode">
  

>   In this video, we discuss Cloudflare One, our Secure Access Service Edge (SASE) platform and how it's been designed to revolutionize the corporate network and enable companies with their Zero Trust strategy. Legacy network design is struggling to address today's challenges of security, performance and monitoring needs. Many IT teams are trying to evolve their corporate network with point solutions and finding the lack of integration and performance an issue.

Reviewer: ```suggestion
  In this video, we discuss Cloudflare One, our Secure Access Service Edge (SASE) platform and how it has been designed to revolutionize the corporate network and enable companies with their Zero Trust strategy. Legacy network design is struggling to address today's challenges of security, performance and monitoring needs. Many IT teams are trying to evolve their corporate network with point solutions and finding the lack of integration and performance an issue.
```

    <Card>
        <Stream
        videoId="a02e5f9e58999d96c3ec9dbb0efb9707"
        videoTitle="video 1"
        thumbnailTimeOrURL="https://pub-d9bf66e086fb4b639107aa52105b49dd.r2.dev/Simon-SASE-thumbnail-test.png"
        moreVideosLink="false"
      />

      **Related content**

>       If you want to dive into detail about modernizing your corporate network with Cloudflare, read here:

Reviewer: ```suggestion
      If you want to dive into detail about modernizing your corporate network with Cloudflare, refer to the following pages:
```

        - [Evolving to a SASE architecture with Cloudflare](/reference-architecture/architectures/sase/)
        - [Network-focused migration from VPN concentrators to Zero Trust Network Access](/reference-architecture/design-guides/network-vpn-migration/)

>         - [Using a zero trust framework to secure SaaS applications](/reference-architecture/design-guides/zero-trust-for-saas/)

Reviewer 2: ```suggestion
        - [Using a Zero Trust framework to secure SaaS applications](/reference-architecture/design-guides/zero-trust-for-saas/)
```

>         - [Building zero trust architecture into your startup](/reference-architecture/design-guides/zero-trust-for-startups/)

Reviewer 2: ```suggestion
        - [Building Zero Trust architecture into your startup](/reference-architecture/design-guides/zero-trust-for-startups/)
```
